core/state_transition.go

```diff
@@ -244,7 +246,7 @@ func (st *StateTransition) buyGas() error {
 	mgval := new(big.Int).SetUint64(st.msg.GasLimit)
 	mgval = mgval.Mul(mgval, st.msg.GasPrice)
 	var l1Cost *big.Int
-	if st.evm.Context.L1CostFunc != nil && !st.msg.SkipAccountChecks {
+	if st.evm.Context.L1CostFunc != nil && !st.msg.SkipAccountChecks && !st.msg.IsMintTokenTx {
 		l1Cost = st.evm.Context.L1CostFunc(st.evm.Context.BlockNumber.Uint64(), st.msg.RollupDataGas, st.msg.IsDepositTx)
```

Comment:
In `NewL1CostFunc()` (`core/types/rollup_l1_cost.go`), do we need to check if the tx is mintTokenTx as well?
e.g.)
```go
if config.Kroma == nil || isDepositTx || isMintTokenTx || rollupDataGas == 0 {
	return nil
}
```